code/game/turfs/simulated/floor.dm

```diff
@@ -147,11 +147,14 @@ var/list/icons_to_ignore_at_floor_init = list("damaged1","damaged2","damaged3","
 			burnt = 0
 			user << "<span class='danger'>You remove the broken plating.</span>"
 		else
-			if(istype(src, /turf/open/floor/wood))
-				user << "<span class='danger'>You forcefully pry off the planks, destroying them in the process.</span>"
+			if (user.a_intent != "harm")
+				if(istype(src, /turf/open/floor/wood))
+					user << "<span class='danger'>You forcefully pry off the planks, destroying them in the process.</span>"
```

Comment:
`You forcefully pry off [src.name], destroying \them in the process.`
note: never tried \them, so test it out to see if it works. not sure where the \ things are anyway